manifest.json

```diff
@@ -33,5 +33,8 @@
     "quick-find-search-links": {
       "description": "Search all links"
     }
-  }
+  },
+  "permissions": [
```

Comment:
Awesome work! Only change requested is to change this to optional permissions. So we'll maintain backward compatibility. https://developer.chrome.com/extensions/permissions The permissions system for extensions isn't ideal but the way it works is understandable. When you add a new permission it prompts the user for the new permissions and if they decide against the new perms the extension is completely disabled.